cgs_vmc/wavefunctions.py

```diff
@@ -387,6 +387,24 @@ def from_hparams(
       fcnn_params['name'] = name
     return cls(**fcnn_params)
 
+  @classmethod
```

Comment:
Some comments and questions:
1) what is the purpose of this method? why wouldn't we just use `from_hparams` class method with modified output_activation?
2) It's not a good idea to have multiple build methods that take similar arguments.